test/integration/armadaserver_controller_e2e_test.go

```diff
@@ -0,0 +1,56 @@
+package integration
+
+import (
+	"io"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var armadaserverYaml = `apiVersion: install.armadaproject.io/v1alpha1
+kind: ArmadaServer
+metadata:
+  labels:
+    app.kubernetes.io/name: armadaserver
+    app.kubernetes.io/instance: armadaserver-sample
+    app.kubernetes.io/part-of: armada-operator
+    app.kubernetes.io/created-by: armada-operator
+  name: armadaserver-e2e
+spec:
```

Comment:
Looking at the CI for integration, all you need to get that to work is to add a Name field to your example or remove the Name field from the config.

Reply:
Name is now removed from the config  - commit d3971fb.